app/jobs/hyku_addons/validate_importer_works_job.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class ValidateImporterWorksJob < ApplicationJob
+  # non_tenant_job
+
+  def perform(account, importer)
+    AccountElevator.switch! account.cname
+    importer.entries.find_each do |entry|
+      next if entry.is_a?(HykuAddons::CsvAdminSetEntry)
```

Comment:
Does this do okay with `HykuAddons::CsvCollectionEntry` objects?
Otherwise we could do:
```suggestion
      next unless entry.is_a?(HykuAddons::CsvEntry)
```